pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -218,7 +223,15 @@ func configureFromProviderConfig(pushCtx *model.PushContext, proxy *model.Proxy,
 			}
 			return otelConfig(serviceCluster, hostname, clusterName)
 		}
-
+		// Providers without any tracing support
+	case *meshconfig.MeshConfig_ExtensionProvider_EnvoyExtAuthzHttp,
```

Comment:
why not use default in case a new provider supported in future

Reply:
The intent was to explicitly list them all so that we can be sure if a new one is added we don't forget to add it here if it needs tracing updates, etc.

However, I should _also_ add `default` since it will crash if we miss.. will do that